spec/models/post_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+describe Post, type: :model do
+  it { is_expected.to have_many :comments }
+
+  it 'is not valid without user providing content' do
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping. (https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)